src/content/1.7/development/webservice/tutorials/4-advanced-use/additional-list-parameters.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Additional list parameters
+weight: 1
+---
+
+# Additional list parameters
+
+Previous tutorials only use simple list that return the IDs of the resources, but you can actually get more details, sort and filter the list.
+
+## Display parameter
+
+You can define which field of each resource you are expecting with the `display` parameter
+
+| Key         | Value               | Result                                              |
+|-------------|---------------------|-----------------------------------------------------|
+| **display** | `full`                | Returns all the fields of the resource            |
+|             | `[field1,field2,...]` | Returns the list of fields specified in the array |
+
+| Result | API call | PHP Webservice lib options |
+|--------|----------|----------------------------|
+| Include *all* fields from the `products` resource | `/api/products/?display=full` | {{< code php >}}$opt = array(
+    'resource' => 'products',
+    'display'  => 'full'
+);{{< /code >}} |
+| Include only the *ID* from the `carriers` resource | `/api/carriers/?display=[id]` | {{< code php >}}$opt = array(
+    'resource' => 'carriers',
+    'display'  => '[id]'
+);{{< /code >}} |
+| Only include the *name* and *value* fields from the `configurations` resource | `/api/configurations/?display=[name,value]` | {{< code php >}}$opt = array(
+    'resource' => 'configurations',
+    'display'  => '[name,value]'
+);{{< /code >}} |
+
+## Filter parameter
+
+You can filter the expected resulted with the `filter` parameter
+
+| Key               | Value                               | Result                                                              |
+|-------------------|-------------------------------------|---------------------------------------------------------------------|
+| **filter[field]** | {{< code >}}[1|5]{{< /code >}} | `OR` operator: list of possible values                              |
+|                   | {{< code >}}[1,10]{{< /code >}}     | `Interval` operator: define interval of possible values             |
+|                   | {{< code >}}[John]{{< /code >}}     | `Literal` value (not case sensitive)                                |
+|                   | {{< code >}}[Jo]%{{< /code >}}      | `Begin` operator: fields begins with the value (not case sensitive) |
+|                   | {{< code >}}%[hn]{{< /code >}}      | `End` operator: fields ends with the value (not case sensitive)     |
+|                   | {{< code >}}%[oh]%{{< /code >}}     | `Contains` operator: fields contains the value (not case sensitive) |
```

Comment:
Use backticks:

```suggestion
| **filter[field]** | `[1|5]`      | `OR` operator: list of possible values                              |
|                   | `[1,10]`     | `Interval` operator: define interval of possible values             |
|                   | `[John]`     | `Literal` value (not case sensitive)                                |
|                   | `[Jo]%`      | `Begin` operator: fields begins with the value (not case sensitive) |
|                   | `%[hn]`      | `End` operator: fields ends with the value (not case sensitive)     |
|                   | `%[oh]%`     | `Contains` operator: fields contains the value (not case sensitive) |
```

Reply:
Actually I had to add this shortcode because the `` notation breaks the table when there special characters like `|` (maybe on others but this one I'm sure) So I tried using an html character notation but it doesn't work in ``

With the shortcode it works fine, it's even more necessary when you need to display pieces of code on multiline inside the table

So in this case I actually only need it for `[1|5]` but I prefered to keep it on other lines for consistency, I can use it on the first line only if you prefer